tests-config-sample.php

```diff
@@ -1,132 +1,68 @@
-	'AUTH_KEY'                => '%%AUTH_KEY%%',
-	'SECURE_AUTH_KEY'         => '%%SECURE_AUTH_KEY%%',
-	'LOGGED_IN_KEY'           => '%%LOGGED_IN_KEY%%',
-	'NONCE_KEY'               => '%%NONCE_KEY%%',
-	'AUTH_SALT'               => '%%AUTH_SALT%%',
-	'SECURE_AUTH_SALT'        => '%%SECURE_AUTH_SALT%%',
-	'LOGGED_IN_SALT'          => '%%LOGGED_IN_SALT%%',
-	'NONCE_SALT'              => '%%NONCE_SALT%%',
-
-	// Security Directives
-	'DISALLOW_FILE_EDIT'      => true,
-	'DISALLOW_FILE_MODS'      => true,
-	'FORCE_SSL_LOGIN'         => false,
-	'FORCE_SSL_ADMIN'         => false,
-
-	// Performance
-	'WP_CACHE'                => true,
-	'WP_MEMORY_LIMIT'         => '96M',
-	'WP_MAX_MEMORY_LIMIT'     => '256M',
-	'EMPTY_TRASH_DAYS'        => 7,
-	'WP_MEMCACHED_KEY_SALT'   => 'tribe',
 
-	// Debug
-	'WP_DEBUG'                => false,
-	'WP_DEBUG_DISPLAY'        => false,
-	'SAVEQUERIES'             => false,
-	'SCRIPT_DEBUG'            => false,
-	'CONCATENATE_SCRIPTS'     => false,
-	'COMPRESS_SCRIPTS'        => false,
-	'COMPRESS_CSS'            => false,
-
-	// Domain Mapping
-	//'SUNRISE'                 => true,
+/*
+ * React dev
+ */
 
-	// Miscellaneous
-	'WP_POST_REVISIONS'       => true,
-	'WP_DEFAULT_THEME'        => 'core'
-);
+define( 'HMR_DEV', false );
 
-// ==============================================================
-// Use defaults array to define constants where applicable
-// ==============================================================
+/*
+ * Debugging
+ */
 
-foreach ( $config_defaults AS $config_default_key => $config_default_value ) {
-	if ( ! defined( $config_default_key ) )
-		define( $config_default_key, $config_default_value );
-}
+define( 'WP_DEBUG', true );
+define( 'SAVEQUERIES', true );
+define( 'WP_DEBUG_DISPLAY', true );
+define( 'SCRIPT_DEBUG', true );
+define( 'WP_CACHE', true );
 
-// ==============================================================
-// Table prefix
-// Change this if you have multiple installs in the same database
-// ==============================================================
+/*
+ * Define a dedicated cache salt for tests to better isolate them
+ * from object caches from other types of request.
+ *
+ * If you're using WPDb for direct database manipulation, the
+ * cache invalidation mechanism might be compromised and you might
+ * face unexpected results in tests that are hard to debug.
+ *
+ * If you'd like to disable object caching completely during tests:
+ * 1 - Replace the line below with "define( 'WP_CACHE_KEY_SALT', md5( microtime( true ) ) );"
+ * 2 - Call "if ( tribe_is_codeception() ) wp_suspend_cache_addition( true );" on a mu-plugin
+ *
+ * Generally speaking, it's a good idea to run tests with cache enabled
+ * because they run faster and you get to test your cache invalidation logic
+ */
+define( 'WP_CACHE_KEY_SALT', 'tests' );
 
-if ( empty( $table_prefix ) )
-	$table_prefix = 'tribe_';
+$GLOBALS['memcached_servers'] = [ [ 'memcached', 11211, ] ];
```

Comment:
We now use redis as the default object-cache.php, should add this in addition to the memcached global

```
/**
 * Redis
 */
$GLOBALS['redis_server'] = [
	'host'     => 'redis',
	'port'     => 6379,
	'auth'     => '',
	'database' => 0,
];
```

Reply:
It seems the default object cache on `master` is still Memcached. `tests-config-sample.php` basically mirrors `local-config-sample.php`, with only a few differences.

We could easily add Redis configuration to `tests-config-sample.php`, but since the default is still Memcached, it could be a bit confusing, don't you think?